src/cascadia/TerminalApp/CascadiaSettingsSerialization.cpp

```diff
@@ -174,6 +174,7 @@ std::unique_ptr<CascadiaSettings> CascadiaSettings::LoadAll()
             // Use 4 spaces to indent instead of \t
             wbuilder.settings_["indentation"] = "    ";
             wbuilder.settings_["enableYAMLCompatibility"] = true; // suppress spaces around colons
+            wbuilder.settings_["allowTrailingCommas_"] = true; // Allow trailing comma in objects and arrays

```

Comment:
wot

this is the builder for the writer that generates the keybinding data blob for ETW

Reply:
don't we need one for .. the .. reader?

Reply:
oh nuts, wait, what?

Reply:
```
void CharReaderBuilder::setDefaults(Json::Value* settings) {
  //! [CharReaderBuilderDefaults]
  (*settings)["collectComments"] = true;
  (*settings)["allowComments"] = true;
  (*settings)["allowTrailingCommas"] = true;
```

https://github.com/open-source-parsers/jsoncpp/pull/1098/files#diff-6af171cad54040cf2fb701a6ece6681bR1953

Reply:
Oh that'll do it. Well. I'll revert the writer changes, but leave the test, since that seems like goodness anyways.